tests/continuous_integration/test_05_network.py

```diff
@@ -82,5 +82,5 @@ def test_command():
         network(args)
 
         df = pd.read_table(fname, sep="\t")
-        assert df.shape[0] == 30690
+        assert df.shape[0] == 30129
```

Comment:
Why is this? Duplicate names?

Reply:
I've checked with the test expression file, and there were some 20 odd duplicate gene names there. 
I didn't check the h5 file, but it seemed plausible enough.